pcb/rules/rule8_2.py

```diff
@@ -9,7 +9,7 @@ class Rule(KLCRule):
     Create the methods check and fix to use with the kicad_mod files.
     """
     def __init__(self, module, args):
-        super(Rule, self).__init__(module, args, 'Rule 6.4', 'For surface-mount devices, footprint anchor is placed in the middle with respect to device lead ends. (IPC-7351).')
+        super(Rule, self).__init__(module, args, 'Rule 8.2', 'For surface-mount devices, footprint anchor is placed in the middle of the footprint (IPC-7351).')
 
     def check(self):
         """
```

Comment:
I think there should be a message output for the violation, e.g. add `self.addMessage("Achor not placed placed in the middle of the footprint (IPC-7351).")` before `return True` in line 35